textdata_reader.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+mimic-cxr 2.0.0 medical report preprocessor for the text that does extraction of findings
+and impression part of the text. Tokenize the sentence, lowercase all the characters and
+remove the word contains non-alphabetic characters
+"""
+import argparse
+import pandas as pd
```

Comment:
you probably need to specify the requirement for pandas in requirements and setup.py